app/src/main/res/values/strings.xml

```diff
@@ -126,6 +126,10 @@
 	<string name="pubkey_import_encrypted_title">"Import encrypted key"</string>
 	<string name="pubkey_import_encrypted_message">"Enter password to decrypt key '%1$s' (%2$s)"</string>
 	<string name="pubkey_import_button">"Import"</string>
+	<string name="pubkey_import_encrypt_key">"Encrypt key for storage"</string>
```

Comment:
The comment before each line should contain a description for translators to know how to translate it.

Reply:
Added comments.

Reply:
Added comments.